sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ChangeFeedQueryImpl.java

```diff
@@ -121,6 +122,17 @@ public Flux<FeedResponse<T>> executeAsync() {
         );
     }
 
+    private void validateContainerRid(String currentContainerRid) {
+        if (!currentContainerRid.equalsIgnoreCase(this.changeFeedState.getContainerRid())) {
+            throw new IllegalStateException(
+                String.format(
+                    "The provided change feed continuation state is for a different container. " +
+                    "ChangeFeedState containerRid: {}, current container {}",
```

Comment:
nit: incorrect formatting placeholders, `%s`

Reply:
fixed, thanks for the catch